packages/apputils-extension/src/themesplugins.ts

```diff
@@ -123,10 +129,58 @@ export const themesPlugin: JupyterFrontEndPlugin<IThemeManager> = {
         if (theme === manager.theme) {
           return;
         }
+        // Disable adaptive theme if users decide to change the theme when adaptive theme is on
+        if (manager.isToggledAdaptiveTheme()) {
+          return manager.toggleAdaptiveTheme();
+        }
         return manager.setTheme(theme);
       }
     });
 
+    commands.addCommand(CommandIDs.changePreferredLightTheme, {
+      label: args => {
+        const theme = args['theme'] as string;
+        const displayName = manager.getDisplayName(theme);
+        return args['isPalette']
+          ? trans.__('Use Theme: %1', displayName)
+          : displayName;
```

Comment:
The label is not specific enough:

```suggestion
        return trans.__('Set %1 as preferred light theme', displayName);
```